lib/src/shared/main/java/com/couchbase/lite/Dictionary.java

```diff
@@ -98,6 +101,7 @@ public int count() {
      * @param key the key.
      * @return the object value or nil.
      */
+    @NonNull
```

Comment:
inconsistent with line 102 NonNull vs return nil

Reply:
Good call!  Fixing!

Reply:
and I think it allows to return null..

Reply:
I don't think so...  `MDict.get` cannot return `null`...

Reply:
fixed

Reply:
I checked .Net and looks like there is chance that getValue can return null.. please double check with @pasin ?

Reply:
I thought getValue method returns object? not MValue..

Reply:
MValue is an Object

Reply:
Look at the code down there in MDict.java and assure yourself that this function cannot return null

Reply:
The return value is nullable from the API spec. The MValue is an internal class and Java shouldn't return the MValue object directly. For example, when the key doesn't exist, the null value should be returned.

Reply:
Certainly not relevant to this PR.  If you have found a bug, please file it.

Reply:
NVM..  I filed it: https://issues.couchbase.com/browse/CBL-542